extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -61,7 +61,7 @@ require([
 	// Everything starts after content and JS
 	win.wgAfterContentAndJS.push(function () {
 		if (floatingRail) {
-			pageLevelParams.add('rrspace', floatingRail.getAvailableSpaceParameter().toString());
+			pageLevelParams.add('ahspace', floatingRail.getArticleHeightParameter().toString());
```

Comment:
What do you think about moving it to https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/AdLogicPageParams.js#L164 - will it be useful info on Mercury?

additional: s/ahspace/ah